docs/application/tizen-studio/native-tools/ui-builder/custom-ui-component.md

```diff
@@ -55,17 +55,17 @@ A custom component requires an EDJ resource file (`[ComponentName].edj`) and an
     | <component> | <state>     | default_value | State  property default value. The default value must be a name attribute defined in the <value> element attributes. | String  | Yes      | -                                    |
     | <state>     | <value>     | name          | Value  property identifier               | String  | Yes      | -                                    |
     | <state>     | <value>     | display_name  | Name  shown on the value property label in the Properties view | String  | No       | name attribute in the <value>element |
-    | <state>     | <value>     | signal        | EDC  signal used to change the UI state at runtime.  For information on using signals, see [Program Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_program_n.htm). | String  | Yes      | -                                    |
+    | <state>     | <value>     | signal        | EDC  signal used to change the UI state at runtime.  For information on using signals, see [Program Block](../native/guides/ui/efl/learn-edc-program.md). | String  | Yes      | -                                    |
     | <state>     | <value>     | source        | User-defined  parameter sent when the EDC signal is triggered | String  | No       | null                                 |
     | <component> | <text>      | name          | Text  property identifier                | String  | Yes      | -                                    |
     | <component> | <text>      | display_name  | Name  shown on the text property label in the Properties view | String  | No       | name attribute in the <text>element  |
     | <component> | <text>      | description   | Text  property description               | String  | No       | null                                 |
-    | <component> | <text>      | part_name     | part block to change when the text property is modified.  For more information, see [Part Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_part_n.htm). | String  | Yes      | -                                    |
+    | <component> | <text>      | part_name     | part block to change when the text property is modified.  For more information, see [Part Block](../native/guides/ui/efl/learn-edc-part.md). | String  | Yes      | -                                    |
```

Comment:
It looks `../../../native/guides/ui/efl/learn-edc-part.md` is correct.

Reply:
done